MPM/mpm_gateway.php

```diff
@@ -173,7 +173,16 @@ public function process_payment($order_id)
 			return array('result' => 'failure');
 		}
 
-		$order->update_status('pending', __('Awaiting payment confirmation', 'MPM'));
+		// We put a banktransfer order 'on-hold'. Woo will deduct stock and won't cancel the order after # minutes.
+		if ($this->id == Mollie_API_Object_Method::BANKTRANSFER)
+		{
+			$order->update_status('on-hold', __('Awaiting payment confirmation', 'MPM'));
+			$order->reduce_order_stock();			
```

Comment:
`reduce_order_stock` is dus niet nodig. Gebeurt bij rapporteren `paid` in `$order->payment_complete()`